tools/pythonpkg/tests/fast/test_expression.py

```diff
@@ -0,0 +1,572 @@
+        col1 = ColumnExpression('a')
+        col2 = ColumnExpression('b')
+        col3 = ColumnExpression('c')
+        expr1 = col1 != col2
+        expr2 = col1 != col3
+        rel2 = rel.select(expr1, expr2)
+        res = rel2.fetchall()
+        assert res == [(True, False)]
+
+    def test_comparison_expressions(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+            select
+                1 as a,
+                2 as b,
+                3 as c,
+                3 as d
+        """
+        )
+        col1 = ColumnExpression('a')
+        col2 = ColumnExpression('b')
+        col3 = ColumnExpression('c')
+        col4 = ColumnExpression('d')
+
+        # Greater than
+        expr1 = col1 > col2
+        expr2 = col2 > col1
+        expr3 = col3 > col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(False, True, False)]
+
+        # Greater than or equal
+        expr1 = col1 >= col2
+        expr2 = col2 >= col1
+        expr3 = col3 >= col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(False, True, True)]
+
+        # Less than
+        expr1 = col1 < col2
+        expr2 = col2 < col1
+        expr3 = col3 < col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(True, False, False)]
+
+        # Less than or equal
+        expr1 = col1 <= col2
+        expr2 = col2 <= col1
+        expr3 = col3 <= col4
+        rel2 = rel.select(expr1, expr2, expr3)
+        res = rel2.fetchall()
+        assert res == [(True, False, True)]
+
+    def test_expression_alias(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+            select 1 as a
+        """
+        )
+        col = ColumnExpression('a')
+        col = col.alias('b')
+
+        rel2 = rel.select(col)
+        assert rel2.columns == ['b']
+
+    def test_star_expression(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+            select
+                1 as a,
+                2 as b
+        """
+        )
+        star = StarExpression()
+        rel2 = rel.select(star)
+        res = rel2.fetchall()
+        assert res == [(1, 2)]
+
+        # With exclude list
+        star = StarExpression(exclude=['a'])
+        rel2 = rel.select(star)
+        res = rel2.fetchall()
+        assert res == [(2,)]
+
+    def test_struct_expression(self):
+        con = duckdb.connect()
+
+        rel = con.sql(
+            """
+            select
+                1 as a,
+                2 as b
+        """
+        )
+
+        col1 = ColumnExpression('a')
+        col2 = ColumnExpression('b')
+        expr = FunctionExpression('struct_pack', col1, col2).alias('struct')
+
+        rel = rel.select(expr)
+        res = rel.fetchall()
+        assert res == [({'a': 1, 'b': 2},)]
+
+    def test_function_expression(self):
```

Comment:
Add tests with some existing functions

Reply:
Also add some tests with aggregates, fine if they throw an error, we're not aiming to support these yet